tests/pytests/test_scf_options.py

```diff
@@ -67,31 +67,31 @@ def test_guess_mix_for_broken_symmetry(inp):
 def test_scf_guess(inp, ref):
     vals = {
         "rhf": {
-            "core": -85.33505416384983,
-            "gwh": -92.48691177199250,
-            "sad": -99.63941801281894,
-            "sadno": -99.77439003262768,
-            "huckel": -99.82915309755559,
-            "modhuckel": -99.80751245771735,
-            "sap": -99.97316521623380,
+            "core": -85.33505416385,
+            "gwh": -92.48691177199,
+            "sad": -100.02909951427,
+            "sadno": -99.60935769112,
+            "huckel": -99.82044374140,
+            "modhuckel": -99.79629341034,
+            "sap": -99.97316521623,
         },
         "uhf": {
-            "core": -88.99202896495355,
-            "gwh": -94.35585144644561,
-            "sad": -99.63941801281894,
-            "sadno": -99.11882081641045,
-            "huckel": -99.20374047108815,
-            "modhuckel": -99.17689049002892,
-            "sap": -99.50144270485751,
+            "core": -88.99202896495,
+            "gwh": -94.35585144645,
+            "sad": -99.34836107817,
```

Comment:
Is it a concern that with the change SAD UHF & ROHF (and often sadno, *huckel) guess energies are _less_ deep?

Reply:
No, the issue is that the test here is FH+, which has orbitals that are more compact than the neutral FH. It is logical that the proper guess for the neutral molecule yields a slightly higher energy for the cation.

Like I mention above, right now the SAD code is placing $n_i^2$ electrons on each orbital that should have a fractional occupation of $n_i$. For hydrogen this means that currently the guess has `0.5` electrons for the hydrogen atom, whereas fluorine probably has 7 electrons distributed evenly on 4 spatial orbitals [no symmetry, this is the key feature in the OpenOrbitalOptimizer guess in #3136] which means that its number of electrons is also wrong by `-0.875`.  Therefore, the current guess is spuriously good for FH+ as it corresponds to a cation with charge 1.375.